backend/src/services/review.ts

```diff
@@ -89,6 +89,14 @@ export async function createAccessRequestReviews(model: ModelDoc, accessRequest:
   await Promise.all(createReviews)
 }
 
+export async function removeAccessRequestReview(accessRequestId: string) {
+  const accessRequestReview = await findReviewForAccessRequest(accessRequestId)
```

Comment:
Did you consider just using a single delete Mongo query rather than two separate Mongo queries?

Reply:
Good thinking. I will refactor the functions to do this.

Reply:
In changing from removing only a single review to removing multiple reviews for a single access request, it is not possible to do this as a single query. A `find`, followed by `deleteMany` will be required to handle multiple reviews.